README.md

```diff
@@ -1,4 +1,9 @@
-# app-gn-embeddable
+# DEPRECATED: app-gn-embeddable
+
+:warning: This repository is no longer maintained and is thus unsupported. 
```

Comment:
I'd use a `> [!WARNING]` [blockquote](https://github.com/orgs/community/discussions/16925) to make it really clear.